docs/_renderer.py

```diff
@@ -1,18 +1,63 @@
 from __future__ import annotations
 
-import griffe.dataclasses as dc  # noqa: F401
-import griffe.docstrings.dataclasses as ds  # noqa: F401
-import griffe.expressions as expr  # noqa: F401
-import quartodoc.ast as qast
+import quartodoc as qd
+import toolz
 from plum import dispatch
-from quartodoc import MdRenderer
 
 
-class Renderer(MdRenderer):
+class Renderer(qd.MdRenderer):
     style = "ibis"
 
     @dispatch
-    def render(self, el: qast.ExampleCode):
-        return f"""```
-{el.value}
-```"""
+    def render(self, el: qd.ast.ExampleCode) -> str:
+        lines = el.value.splitlines()
+
+        result = []
+
+        prompt = ">>> "
+        continuation = "... "
+
+        skip_doctest = "doctest: +SKIP"
+        expect_failure = "quartodoc: +EXPECTED_FAILURE"
+        quartodoc_skip_doctest = "quartodoc: +SKIP"
```

Comment:
Made an issue upstream to track this: https://github.com/machow/quartodoc/issues/266